### PR TITLE
P0009R13: Incorporate p2299r3 changes

### PR DESCRIPTION
**Pull request by mhoemmen**
Incorporate changes proposed by P2299R3, and make a few minor fixes.

### CODE REVIEW

**Review by crtrott — APPROVED** [review]

File: P0009/P0009r12.md [hunk 1]
```diff
@@ -742,8 +785,8 @@ shall determine._
 >  _Make the following changes to 22.7.1 <b>[views.general]</b>_,
 
 [1]{.pnum} The header `<span>` defines the view span. 
-[The header `<mdspan>` defines the view `basic_mdspan`, the type alias 
-`mdspan`, and other facilities for interacting with these views.]{.add}
+[The header `<mdspan>` defines the view `mdspan`
```
Comment (dalg24) [inline P0009/P0009r12.md:788]: should it be saying "the class template `mdspan` and other facility for interacting with these non-owning multidimensional views."

**Review by dalg24 — COMMENTED** [review]
LGTM

**Review by nliber — APPROVED** [review]

File: P0009/P0009r12.md (hunk 1 above)
Comment (crtrott) [reply P0009/P0009r12.md:788]: agree with damien

Comment (mhoemmen) [reply P0009/P0009r12.md]: I implemented this fix -- thanks Damien and Christian!

File: P0009/P0009r12.md [hunk 2]
```diff
@@ -2202,21 +2282,21 @@ define the values of `first[r]` and `last[r]` as follows:
...
 // Initialize the span
 for(int i0=0; i0<a.extent(0); i0++)
   for(int i1=0; i1<a.extent(1); i1++)
     for(int i2=0; i2<a.extent(2); i2++)
-      a(i0,i1,i2) = 10000*i0+100*i1+i2;
+      a(i0,i1,i2) = 10000*i0 + 100*i1 + i2;
 
 // Create Subspan
-auto a_sub = submdspan(a,1,pair<int,int>(4,6),pair<int,int>(1,6));
+auto a_sub = submdspan(a, 1, pair<int,int>(4,6), pair<int,int>(1,6));
```
Comment (keryell) [inline P0009/P0009r12.md:2299]: Now we have CTAD, could we simplify the code, like
```suggestion
auto a_sub = submdspan(a, 1, pair { 4, 6 }, pair { 1, 6 });
```
?
Could it be possible to reach
```c++
auto a_sub = submdspan(a, 1, { 4, 6 }, { 1, 6 });
```
?

Comment (mhoemmen) [reply P0009/P0009r12.md:2299]: [CTAD works fine](https://godbolt.org/z/rvbTcqT8e), but unadorned curly braces do not work.

I'll be happy to make this change, but please remember that examples are nonnormative.

Comment (crtrott) [reply P0009/P0009r12.md]: hi, one of the goals was to leave this C++14 compliant for now. Largely due to limitations in some of the customer software stacks (like LLNL Sierra, the top 3 HPC machine in the world can't fully use C++17 yet ...).

**Pull request merged**